packages/modules/src/modules/banners/momentTemplate/settings.ts

```diff
@@ -29,6 +29,7 @@ export interface TickerStylingSettings {
 
 export interface HeaderSettings {
     textColour: string;
+    textHeight?: number;
```

Comment:
Is this needed?

Reply:
Originally Jade asked for a height change and then was not needed. Thought might come in handy in future but your right its not needed (and suppose is just extra code). Ok if I remove?